lib/server.js

```diff
@@ -102,9 +102,9 @@ function parsePOST(req) {
 
 function logResponse(opts, response, startTime) {
     const latency = Date.now() - startTime;
-    let logLevel = 'trace/request';
+    let logLevel = 'info/request';
```

Comment:
Right, but

```json
"res_status": 200,
    "res_headers_cache-control": "s-maxage=1209600, max-age=0, must-revalidate",
    "res_headers_access-control-allow-origin": "*",
    "res_headers_access-control-allow-headers": "accept, content-type, cache-control, accept-language, api-user-agent, if-match, if-modified-since, if-none-match, dnt, accept-encoding",
    "res_headers_access-control-expose-headers": "etag",
    "res_headers_x-xss-protection": "1; mode=block",
    "res_headers_x-content-type-options": "nosniff",
    "res_headers_x-frame-options": "SAMEORIGIN",
    "res_headers_content-security-policy": "default-src 'none'; frame-ancestors 'none'",
    "res_headers_x-content-security-policy": "default-src 'none'; frame-ancestors 'none'",
    "res_headers_x-webkit-csp": "default-src 'none'; frame-ancestors 'none'",
    "res_headers_etag": "\"658564842/4b17378e-0801-11e7-908c-51c361a58169\"",
    "res_headers_content-type": "application/json; charset=utf-8; profile=\"https://www.mediawiki.org/wiki/Specs/mobile-sections/0.9.1\"",
    "res_headers_vary": "Accept-Encoding",
    "res_headers_date": "Mon, 13 Mar 2017 15:25:30 GMT",
    "res_headers_content-location": "https://en.wikipedia.org/api/rest_v1/page/mobile-sections/User_talk%3APinewood~enwiki?redirect=false",
    "res_headers_access-control-allow-methods": "GET",
    "res_headers_x-request-id": "4cd3e4f8-5108-4950-8f74-137a0e5956e9",
    "root_req_method": "get",
    "root_req_uri": "/en.wikipedia.org/v1/page/mobile-sections/User_talk%3APinewood~enwiki?redirect=false",
    "root_req_headers_cache-control": "no-cache",
    "root_req_headers_content-length": "0",
    "root_req_headers_user-agent": "ChangePropagation/WMF",
    "root_req_headers_x-client-ip": "::ffff:10.64.48.29",
    "root_req_headers_x-forwarded-for": "::ffff:10.64.48.29",
    "root_req_headers_x-request-id": "4cd3e4f8-5108-4950-8f74-137a0e5956e9",
    "root_req_headers_x-request-class": "internal_update",
    "root_req_headers_x-triggered-by": "mediawiki.revision-create:https://en.wikipedia.org/wiki/Template:SockpuppetCheckuser,change-prop.transcludes.resource-change:https://en.wikipedia.org/wiki/User_talk%3APinewood~enwiki,change-prop.transcludes.resource-change:http://en.wikipedia.org/api/rest_v1/page/html/User_talk%3APinewood~enwiki",
```

looks a bit too verbose too me.